<launch>
    <rosparam file="$(find coffee_serving_robot)/config/table_locations.yaml" />

    <!--node pkg="coffee_serving_robot" type="goal_nav.py" name="goal_nav" output="screen" /-->
    <node pkg="coffee_serving_robot" type="goal_serve.py" name="goal_serve" output="screen" />
    <node pkg="coffee_serving_robot" type="keyboard_control.py" name="keyboard_control" output="screen" />
    <node pkg="coffee_serving_robot" type="set_table_locations.py" name="set_table_locations" output="screen" />
    <node pkg="coffee_serving_robot" type="mobile_set_table.py" name="mobile_set_table" output="screen" />
    <node pkg="sound_play" type="soundplay_node.py" name="sound_play_node" output="screen" />
    <node pkg="coffee_serving_robot" type="play_audio.py" name="play_audio" output="screen" />
</launch>
